src/resources/main.js

```diff
@@ -28,16 +28,17 @@ function findMatchingParents(element, selector) {
 
 // Based on document.location, open the necessary tree buttons
 function openTreeFromPath() {
-  const element = treeMap[window.location.pathname];
+  const element = document.querySelector("nav a[aria-current=page]");
```

Comment:
Using the DOM to hold the "tree" instead of us creating it manually is a great idea. Since we're a largely static website with not a lot of dynamic content, I believe this doesn't incur a large performance penalty, compared to what we did before.